docs/perf-tests-region-scheduling.md

```
@@ -0,0 +1,74 @@
+There are performance tests that are triggered on different schedules:
+- Weekly
+- Once per 3 weeks
+- Monthly
+- On every release (unpredictable timing)
+
+We need to assign regions so that tests do not overlap in the same region at the same time.
+Release tests may overlap with monthly triggered tests, but NOT with weekly or 3-weekly tests.
+
+## Original Assignment (before change)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)